app/controllers/providers/allocations_controller.rb

```diff
@@ -72,6 +83,10 @@ def build_recruitment_cycle
       @recruitment_cycle = RecruitmentCycle.find(cycle_year).first
     end
 
+    def build_allocation
```

Comment:
```suggestion
    def allocation
      @_allocation ||= Allocation.find(params["id"]).first
    end
```

Also... Does `.find` return a collection? 😞